ajax/libs/jquery.terminal/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "jquery.terminal",
+  "npmName": "jquery.terminal",
+  "npmFileMap": [
+    {
+      "basePath": "js",
+      "files": [
+        "**/*"
```

Comment:
Should not use `"**/*"` in non-dist folder since you can not make sure it's for distribution.